lib/dice_bag/available_templates.rb

```diff
@@ -16,6 +16,23 @@ def templates_location
 
     class << self
 
+      def gem_specs
+        @gem_specs ||= begin
+          Gem::Specification.sort.each_with_object({}) do |spec, hsh|
+            hsh[spec.name] = spec.full_gem_path
+          end
+        end
+      end
+
+      def checker_within_given_gems?(checker, gem_names)
+        return true if gem_names.empty?
```

Comment:
umm, maybe take this out of here and make it explicit in the place calling this? (i.e. you always a non-empty array as parameter)